ci/test.sh

```diff
@@ -31,7 +31,9 @@ main() {
     export QEMU_STRACE=1
 
     # ensure we have the proper toolchain and optional rust flags
-    export CROSS=("${PROJECT_HOME}/target/debug/cross")
+    local cross_path
+    cross_path=$(binary_path cross "${PROJECT_HOME}" debug)
+    export CROSS=("${cross_path}")
```

Comment:
why the intermediary variable?

Reply:
Ah this was originally due to changes in using it for both `cross` and `cross-util`, which I quickly realized made absolutely no sense.

Reply:
this needs to be changed in other places as well where this was added 👍🏼

Reply:
I believe it was changed everywhere, as that was the only instance of it.